scraper/keldoc/keldoc_center_scrap.py

```diff
@@ -0,0 +1,178 @@
+import csv
+import json
+import multiprocessing
+import os
+from typing import List, Optional
+
+import httpx
+
+from scraper.doctolib.doctolib_center_scrap import doctolib_urlify
+from utils.vmd_config import get_conf_platform, get_conf_inputs
+from utils.vmd_logger import get_logger
+
+KELDOC_CONF = get_conf_platform("keldoc")
+KELDOC_API = KELDOC_CONF.get("api", {})
+SCRAPER_CONF = KELDOC_CONF.get("center_scraper", {})
+BASE_URL = KELDOC_API.get("scraper")
+
+timeout = httpx.Timeout(KELDOC_CONF.get("timeout", 25), connect=KELDOC_CONF.get("timeout", 25))
+KELDOC_ENABLED = KELDOC_CONF.get("enabled", False)
+KELDOC_HEADERS = {
+    "User-Agent": os.environ.get("KELDOC_API_KEY", ""),
+}
+session = httpx.Client(timeout=timeout, headers=KELDOC_HEADERS)
+logger = get_logger()
+
+
+def run_departement_scrap(departement: str) -> list:
+    logger.info(f"[Keldoc centers] Parsing pages of departement {departement} through department SEO link")
+    centers_departements = parse_pages_departement(departement)
+    if centers_departements == 0:
```

Comment:
`parse_pages_department` returns a list, so this should be: `if not centers_departments:`

Reply:
En effet, ce bout vient du scraper Doctolib, ça ne devrait pas être comme ça.